trustmanager/x509utils.go

```diff
@@ -233,6 +233,10 @@ func ValidateCertificate(c *x509.Certificate) error {
 	if (tomorrow).Before(c.NotBefore) || now.After(c.NotAfter) {
 		return fmt.Errorf("certificate is expired")
 	}
+	// If this certificate is expiring within 6 months, put out a warning
+	if (c.NotAfter).Before(time.Now().AddDate(0, 6, 0)) {
+		logrus.Warn("certificate is expiring within 6 months")
```

Comment:
what other info can we include to make it clear _which_ certificate is expiring?